lib/fetchGlobalID.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Treasure Global ID
+ */
```

Comment:
Would it be cleaner to implement this as a new plug-in (since plug-ins get [added to the prototype](https://github.com/treasure-data/td-js-sdk/blob/0266bba1a26f551096ff1750b49f6f2eddf4c2e7/lib/treasure.js#L87-L94) in the same way)?

I guess that might be a little weird since this function depends on the `/plugins/track.js` plug-ing (b'c of `getCookie` / `setCookie`.) Just thinking out loud.